server/sonar-web/src/main/js/components/common/MultiSelect.tsx

```diff
@@ -23,21 +23,22 @@ import MultiSelectOption from './MultiSelectOption';
 import SearchBox from '../controls/SearchBox';
 
 interface Props {
-  selectedElements: Array<string>;
-  elements: Array<string>;
+  elements: string[];
   listSize?: number;
+  loading?: boolean;
```

Comment:
I wonder if it'd better to have `loading` inside the component's state. If we change `onSearch` prop to return a Promise `onSearch: (query: string) => Promise<void>`, we could use this promise to change the `loading` flag. WDYT?

Reply:
Yes I think it would be, the only thing is the initial value of `loading`, since we receive the elements from outside, the initial `onSearch` is not called from inside that component currently